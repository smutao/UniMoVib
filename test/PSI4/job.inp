A test job of PSI.

 $contrl
   qcprog="psi"
 $end

 $qcdata
   fchk="h2o.out"
 $end
