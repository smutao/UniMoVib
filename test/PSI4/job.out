 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.1.1, Jan 14, 2018
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      PSI4
 FCHK file:       h2o.out


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    O       8            0.00000000    0.00000000   -0.06770568           15.99491500
     2    H       1            0.00000000   -0.75631229    0.53726912            1.00782500
     3    H       1            0.00000000    0.75631229    0.53726912            1.00782500
 ------------------------------------------------------------------------------------------


 <<< SYMMETRY >>>
 Molecular Point Group (Z)   :   C2v 
 Molecular Point Group (Z+M) :   C2v 


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm**-1), reduced masses (AMU), force constants (mDyn/A)

                                                   1                                 2                                 3
          Irreps                                  A1                                A1                                B2
     Frequencies                           1658.9557                         3753.4042                         3854.5634
  Reduced masses                              1.0820                            1.0458                            1.0800
 Force constants                              1.7544                            8.6808                            9.4546
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.00000   0.00000   0.07034       0.00000  -0.00000   0.05036       0.00000   0.06942   0.00000
           2   1         0.00000  -0.43128  -0.55814       0.00000   0.58229  -0.39959       0.00000  -0.55085   0.44063
           3   1         0.00000   0.43128  -0.55814       0.00000  -0.58229  -0.39959       0.00000  -0.55085  -0.44063

 Results of translations and rotations:

                                                   1                                 2                                 3
          Irreps                                  B1                                B2                                A1
     Frequencies                              0.0000                           -0.0000                           -0.0001
  Reduced masses                              6.0035                            6.0035                            6.0035
 Force constants                              0.0000                           -0.0000                           -0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735
           2   1         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735
           3   1         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735

                                                   4                                 5                                 6
          Irreps                                  B1                                A2                                B2
     Frequencies                              0.0000                            0.0000                            0.0001
  Reduced masses                              1.1259                            1.0078                            1.0476
 Force constants                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8        -0.08876   0.00000   0.00000       0.00000   0.00000  -0.00000       0.00000   0.05154   0.00000
           2   1         0.70432   0.00000   0.00000       0.70711   0.00000   0.00000      -0.00000  -0.40896  -0.57569
           3   1         0.70432   0.00000   0.00000      -0.70711   0.00000  -0.00000       0.00000  -0.40896   0.57569


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Molecular mass            :        18.010565    AMU
 Elec. total energy        :         0.000000    Hartree
 Scale factor of Frequency :         1.000000
 Rot. symmetry number      :         2
 The C2v  point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                  2.339602            4.117325            6.456928
           X                         0.000000            0.000000            1.000000
           Y                         1.000000            0.000000            0.000000
           Z                         0.000000            1.000000            0.000000

 Rotational temperatures            37.020742           21.036427           13.414091    Kelvin
 Rot. constants A, B, C             25.730735           14.621067            9.323271    cm**-1
                                   771.388031          438.328548          279.504639    GHz


 #   1    Temperature =       298.15000 Kelvin         Pressure =         1.00000 Atm
 ====================================================================================

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.021112           13.247732
 Thermal correction to Energy               :            0.023947           15.026772
 Thermal correction to Enthalpy             :            0.024891           15.619258
 Thermal correction to Gibbs Free Energy    :            0.003442            2.159666
 ====================================================================================


 UniMoVib job terminated correctly! Sun Jan 14 13:30:25 2018
